client/ayon_core/hosts/max/plugins/publish/validate_model_name.py

```diff
@@ -104,7 +104,7 @@ def invalid_name(cls, instance, obj):
         compare = {
             "project": instance.context.data["projectName"],
             "asset": instance.context.data["folderPath"],
-            "subset": instance.context.data["subset"],
+            "subset": instance.name,
```

Comment:
Wait a second - this taking from `context` is wrong to begin with. It should match the `instance` - not the `context`

Also, if `instance.data["subset"]` does not exist then likely in AYON it is `instance.data["product"]` or `instance.data["productName"]` that you should be matching against, not `instance.name` - there is no requirement for the instance name to be or match the product (== subset in OpenPype).

Reply:
fixed